python/semantic_kernel/connectors/memory/azure_cognitive_search/azure_cognitive_search_memory_store.py

```diff
@@ -8,7 +8,7 @@
 from azure.core.exceptions import ResourceNotFoundError
 from azure.search.documents.indexes.aio import SearchIndexClient
 from azure.search.documents.indexes.models import (
-    HnswVectorSearchAlgorithmConfiguration,
+    HnswAlgorithmConfiguration,
```

Comment:
The [unit tests](https://github.com/microsoft/semantic-kernel/actions/runs/8021982266/job/21921710484?pr=5130#step:5:32) are failing for Python 3.8.